### ISSUE TITLE
Confusion about datasets.

### ISSUE THREAD

**Issue by benjpau** [yohanjo/tacl_arg_rel#2]
Thanks for such excellent work.
I have some questions about your dataset.
First, the "kialo_causal_pairs-bi.csv" file contains 91303 samples, but there are only 71314 samples in "kialo_causal_pairs-bi-split.csv", which does not match. The same issue arises in "kialo_causal_pairs-neu.csv" and "kialo_causal_pairs-neu-split.csv", "kialo_normative_pairs-bi.csv" and "kialo_normative_pairs-bi-split.csv", and "kialo_normative_pairs-neu.csv" and "kialo_normative_pairs-neu-split.csv". Can you explain this problem? Did you use only the samples specified in the "\*-split.csv" file in your experiment? Should we follow the "\*-split.csv" file in our experiment?

Second, 
for the kialo normative dataset in Table 4 of your paper: 
According to Table 4, there should be 14246 (6623+7623) (Sup or Att) samples in the test set, but in "kialo_normative_pairs-bi-split.csv", only 14221 test samples are specified.
for the debate normative dataset in Table 4 of your paper: 
According to Table 4, there should be 11100 (6598+4502) (Sup or Att) samples in the train set, but in "debate_normative_pairs-bi-split.csv", only 9120 train sample sare specified.
Could you please explain these questions? Otherwise, it will be hard to follow your work.

**Comment by yohanjo** [comment]
Regarding the first issue:
The datasets have been cleaned up.

Regarding the second issue:
In Table 4, for Normative-sup, the number for Kialo-Test should be 6,598 and Debatepedia-Fit should be 4,618.

Thanks for pointing out the issues.
